docs/usage/SYNC_OPTIONS.md

```diff
@@ -0,0 +1,62 @@
+# Sync Options
+
+#### `errorCallBack`
+a callback that is called whenever an error event occurs during the sync process.
+
+#### `warningCallBack` 
+a callback that is called whenever a warning event occurs during the sync process.
+
+#### `beforeUpdateCallback`
+a filter function which can be applied on a generated list of update actions. It allows the user to intercept 
+**_update_** actions just before they are sent to CTP API.
+
+#### `beforeCreateCallback`
+a filter function which can be applied on a draft before a request to create it on CTP is issued. It allows the 
```

Comment:
```suggestion
a filter function, which can be applied on a draft before a request to create it on CTP is issued. It allows the 
```

Reply:
Is that `is issued` in the end redundant? Can't get the meaning of it.

Reply:
2913f35